deepface/DeepFace.py

```diff
@@ -231,13 +234,13 @@ def verify(
 
 
 def analyze(
-    img_path,
-    actions=("emotion", "age", "gender", "race"),
-    enforce_detection=True,
-    detector_backend="opencv",
-    align=True,
-    silent=False,
-):
+    img_path: Union[str, np.ndarray],,
```

Comment:
Double comma at the end of the line breaks the code

Reply:
Ah, My eyes missed that. 
Apologies! Fixed now